Raspberry Pi/RaspberrySide/DistanceHelper.py

```diff
@@ -48,7 +48,7 @@ def getYRobotPosition(self,robotObject):
         '''
 
         return sin(radians(robotObject.angle))*robotObject.distanceFromTower
-    # MARK: move those to angle helper:
+    # TODO: move those to angle helper:
```

Comment:
It's also present in the other pull request. I'm merging this one and writing up on the second PR.